tests.py

```diff
@@ -206,7 +206,30 @@ def test_circle_group(self):
         for x, y in target_group:
             group = capture_group(x, y, board)
             self.assertEqual(sorted(group), sorted(target_group))
-
+    def test_game_not_final(self):
+        board=np.array(
+                    [[1, 0,-1, 0, 1, 1, 1, 1, 1],
+                     [1, 0, 1, 1, 1, 1, 1, 1, 1],
+                     [1, 1, 1, 1, 1, 1, 1, 1, 1],
+                     [1, 1, 0, 1, 1, 1, 1, 1, 1],
+                     [1, 1, 1, 1, 1, 1, 1, 1, 1],
+                     [1, 1, 1, 1, 1, 1,-1,-1,-1],
+                     [1, 1, 1, 1, 1, 1,-1,-1,-1],
+                     [1, 1, 1, 1, 1, 1,-1,-1, 0],
+                     [1, 1, 1, 1, 1,-1, 0, 0, 0]])
```

Comment:
Is the last 0 the place on the board that causes game_final to return False (because it has no adjacent stones ?) Just making sure I understand the intent of the function.